test/unittests/t_util.cc

```diff
@@ -407,6 +407,8 @@ TEST_F(T_Util, GetFileSystemInfo) {
   EXPECT_EQ(kFsTypeProc, fs_info.type);
   fs_info = GetFileSystemInfo("/");
   EXPECT_EQ(kFsTypeUnknown, fs_info.type);
+  fs_info = GetFileSystemInfo("/run");
```

Comment:
That only works on Linux, and perhaps not on all distributions.  I'd check if `/run` exists first.

Reply:
seeing that this also fails on ubuntu i guess we either find some better tmpfs that is always there or we just remove the test?

Reply:
Probably best to remove it then.  It's a pity but it's tough to test it reliably without being root.